nmrfx-core/src/main/java/org/nmrfx/chemistry/MoleculeBase.java

```diff
@@ -1084,17 +1084,17 @@ public void writeXYZToPDB(String fileName, int whichStruct) throws IOException {
                         lastAtom = atom;
                     }
                 }
-                out.print(lastAtom.spatialSet.toTERString(i + 1) + "\n");
-
-                bondList.forEach((bAtom) -> {
+                if (lastAtom != null) {
+                    out.print(lastAtom.spatialSet.toTERString(i + 1) + "\n");
+                }
+                bondList.forEach(bAtom -> {
```

Comment:
lastAtom should only be null if there are no atoms in the MoleculeBase instance, or no atoms spatial sets were able to be converted to a string. Is it okay to leave this last line off of the file in that case?